public/index.html

```diff
@@ -16,7 +16,8 @@
 		<script type="text/javascript">
 		window.onload = function() {
 			if (document.getElementById('app')) {
-				document.writeln("Sorry, this website requires a modern browser. Please update your browser or use Google Chrome or Mozilla Firefox as alternative.");
+				document.writeln("<h1>Broken openEO Web Editor</h1>");
+				document.writeln("<p>The openEO Web Editor web application failed to load correctly. This could be caused by a server-side problem or a failed browser requirement at your end.</p>");
```

Comment:
```suggestion
				document.writeln("<p>The openEO Web Editor web application failed to load correctly. This could be caused by a server-side problem or a failed browser requirement at your end. Please clear the browser cache and try again. If it still doesn't work,please try using the latest version of Firefox, Chrome, Safari or Edge.</p>");
```

Reply:
> Please clear the browser cache

I assume this is to make sure the user gets the latest js/css assets. I think it's easier advice the user to "hard refresh" the page (e.g. on Windows usually CTRL+F5, on mac CMD+SHIFT+R).

> If it still doesn't work, please try using the latest version of Firefox, Chrome, Safari or Edge

The problem with this statement (and why I avoided it) is that we saw that users were getting quite confused to read that when they were already using one of these mentioned browsers. I think that the sheer majority of  users will already use a recent enough browser anyway. The urgency of aggressively pushing users to modern (e.g. non-IE) browsers is mostly behind us nowadays (I hope). In that sense it feels counter-productive here to trick people into  installing another browser and I would make that statement a bit more vague to prevent that, e.g.:

> If it still doesn't work, consider using another browser.

in summary something like:

```suggestion
				document.writeln("<p>The openEO Web Editor web application failed to load correctly. This could be caused by a server-side problem or a caching/browser issue your end. Try to hard-refresh the page (e.g. CTRL+F5 on Windows or CMD+SHIFT+R on macOS) or use another browser.</p>");
```